pims/cine.py

```diff
@@ -668,3 +887,20 @@ def _sixteen2twelve(b):
         a[k+2:k2:3] = ((b1 & 0x0FF) << 0)
 
     return a
+
+
+def _remove_trailing_x00(dic):
+    """Remove binary null character b'\x00' in dictionary dic entries."""
```

Comment:
This converts bytes to str as well, right? Maybe good to document that

Reply:
Yes, it does. I updated the docstring.